CHANGELOG

```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Initialize new Vue.js project in `zimui` subfolder
 - Update dependencies in pyproject.toml (pydantic, pyhumps, python-slugify)
 - Update scraper to generate JSON files for `zimui` (#212)
+- Remove template files (home.html, article.html) and `make_html_files` method in scraper.py
```

Comment:
Please be more specific with something like `Remove old UI files and methods: template files (home.html, article.html) and `make_html_files` method in scraper.py`

Reply:
Changed in [9981785](https://github.com/openzim/youtube/pull/219/commits/99817858463334030f4807a0ec5f975ac63189c8)